src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,42 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+interface Props {
+  selectedTodo: Todo | null;
+  closeTodo: () => void;
+}
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, closeTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isError, setIsError] = useState(false);
+
+  const loadData = async () => {
+    try {
+      const currentUser = await getUser(selectedTodo?.userId || 0);
+
+      setUser(currentUser);
+    } catch {
+      setIsError(true);
+    }
+  };
+
+  useEffect(() => {
+    loadData();
+  }, [selectedTodo]);
+
+  const isCompleted = selectedTodo?.completed
+    ? 'Done'
+    : 'Planned';
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
+      {!user ? (
```

Comment:
Add a boolean `useState` which will show if it loading or not. 

```suggestion
      {isLoading ? (
```